firebase-database/src/main/java/com/google/firebase/database/core/view/View.java

```diff
@@ -101,6 +101,8 @@ public Node getCompleteServerCache(Path path) {
     if (cache != null) {
       // If this isn't a "loadsAllData" view, then cache isn't actually a complete cache and
       // we need to see if it contains the child we're interested in.
+      // Clarify: just because this is a loadsAllData() doesn't mean that it'll contain the child,
+      // right?
```

Comment:
I think that's right, and if there is no such child, `cache.getChild(path)` will return `EmptyNode.empty()`.